index.js

```diff
@@ -39,7 +39,7 @@ function resolveUserAgent(uaString) {
 
   // Case A: For Safari, Chrome and others browsers
   // that report as Safari after stripping tags
-  if (parsedUA.family.includes('Safari')) {
```

Comment:
This if statement is what was causing desktop Safari browsers to be reported as iOS browsers.